### PR TITLE
authorization fixes

### PR DESCRIPTION
**Pull request by bboozzoo**
Some fixes for issues found during integration.

@kacf @pasinskim

### CODE REVIEW

**Comment by kacf** [comment]
Looks good to me (but I'll let @pasinskim have a look too).

**Comment by bboozzoo** [comment]
@pasinskim ping

File: client_auth.go [hunk 1]
```diff
@@ -58,6 +58,12 @@ func (u *AuthClient) Request(server string, dataSrc AuthDataMessenger) ([]byte,
 	}
 	defer rsp.Body.Close()
 
+	log.Debugf("got response: %v", rsp)
```
Comment (pasinskim) [inline client_auth.go]: Maybe `got response for authentication request: `?

**Comment by pasinskim** [comment]
Looks OK.

**Pull request merged**